### ISSUE TITLE
Analyze test set with omicron

### ISSUE THREAD

**Issue by EthanMarx** [ML4GW/BBHNet#167]
In our initial runs, we have seen more Livingston events causing false alarms, specifically at later times in the testing set. 

There are a couple entangled hypotheses:

1. There happen to be more glitches at Livingston during this time
2. We have introduced some bias that causes our network to perform worse on Livingston 
3. We perform worse on these glitches since they are farther away in time from our training set

If we run omicron over the testing dataset and compare the trigger rates to our networks false alarms,
we can hopefully disentangle these three possibilities.
